### PR TITLE
Fix some typos so Git Workflow reads better

### PR DESCRIPTION
**Pull request by DJA89**
Just some things that I _believe_ can be improved. If any of my changes isn't correct I appreciate the feedback!

### CODE REVIEW

File: git/README.md [hunk 1]
```diff
@@ -55,25 +55,25 @@ This branch is the main one because it reflects the work done by the whole team
...
+This branch has all the features ready to be tested by the QA team.
 
 If the project has CD configured, every time something is merged into this branch the code will be automatically deployed to the QA environment.
 
 * **Origin:** *master* (in the initial project set up this branch is created from master)
 * **Destination:** *staging*
 * **Use case:** Integration of the work done by the development team ready for QA team to test.
 
-**_Note_**: *If a feature merged into QA branch is not ready to be promoted to Staging branch, needed adjustments (removing the feature/disabling access to it/etc) should be made in Develop branch, followed by promoting the code to QA branch to continue the flow.*
+**_Note_**: *If a feature merged into the QA branch is not ready to be promoted to the Staging branch, needed adjustments (removing the feature/disabling access to it/etc) should be made in the Develop branch, followed by promoting the code to the QA branch to continue the flow.*
```
Comment (aguscha333) [inline git/README.md:76]:
```suggestion
**_Note_**: *If a feature merged into the QA branch is not ready to be promoted to the Staging branch, then all needed changes (like: fixes, removing the feature, disabling access to it, etc) should be made in the Develop branch, followed by promoting those changes to the QA branch to continue the flow.*
```

File: git/README.md [hunk 2]
```diff
@@ -136,7 +136,7 @@ Changes in this branch will be reflected in develop after reviewed and approved
 
 If there is a change of any kind in the feature branches in a commit (*git amend*) or in the history (*git rebase*), a new *demo* branch can be created to reflect the change or it can just be merged in the existing branch.
 
-**_Note_**: *In spite of the fact that this branch has as a main purpose showing the progress in a review, it is not advised to send a version from that branch. In these cases you should talk to project or company referents.*
+**_Note_**: *In spite of the fact that this branch's main purpose is showing the progress in a review, it is not advised to send a version from that branch. In these cases you should talk to project or company referents.*
```
Comment (aguscha333) [inline git/README.md:139]: `In spite of` sounds too formal and out of place
```suggestion
**_Note_**: *Despite the fact that this branch's main purpose is showing the progress in a review, it is not advised to send a version from that branch. In these cases you should talk to project or company referents.*
```

**Review by aguscha333 — COMMENTED** [review]
🚀 thanks for the improvements

**Review by CamilaMoscatelli — APPROVED** [review]

**Review by gonzaM — APPROVED** [review]
🙌 👏

**Pull request merged**